### PR TITLE
Fix #136 Dot after return consistency

### LINKED ISSUE #136: Fix Linting Issues
This is a large meta-issue that has many problems. The problems should be divided up.

- [x] Uses do: instead of collect: or select: (Weight: 4) @BenBals #138 
- [x] *Dot After Return Consistency* (Weight: 3) @robertkirsten #150 
      _(Please also check what the style guide says about last-line-dots without a return statement)_
- [x] ifTrue:/ifFalse returns instead of and:/or: (Weight: 1) @phlprcks -> was already refactored a while ago
- [x] intensive Coupling (Weight: 7) @sinnaj-r @BenBals #138
- [x] Law of Demeter (Weight: 9) @sinnaj-r @BenBals  #138
- [x] Long Methods (Weight: 4) @sinnaj-r #139
- [x] Sends questionable message (Weight: 1) @kej-jay 
- [ ] Sends add:/remove: to external collection (Weight: 1) @BenBals #146 
- [x] excessive number of arguments (Weight: 1) @phlprcks -> see comment below
- [x] Methods implemented but not sent (Weight: 1) @phlprcks #160 
- [x] Inconsistent method classification (Weight: 3, easy) @phlprcks -> see comment below
- [x] Temporary Variables not read and written (Weight: 3) @robertkirsten #153 
- [x] Instance Variable not read and written (Weight: 2) @robertkirsten #153 
- [x] Refers to class name instead of "self class" (Weight: 0.1) @kej-jay 
- [x] Unnecessary assignment to a temporary variable (Weight: 1) @robertkirsten #153
- [x] Unreferenced Variable (Weight: 3) @robertkirsten  #158 
	- The weight is so high, because the linter does not tell you which one (as far as I can tell)

| Team Member | Points |
|--|--|
| @sinnaj-r | 20 |
| @robertkirsten | 14 |
| @phlprcks | 6 |
| @kej-jay | 1.1 |
| @BenBals | 21 |

The weights are my subjective, imperfect measure of how time consuming and/or difficult a refactoring is as well as how much risk for escalation of time is involved. I suggest that each pair take on roughly equal amounts of weight. I welcome changes and more linting problems to be added.

This does not invalidate the "boy scout rule" that when you see one of these problems in the code you *must fix it immediatly*. That holds especially for simple refactorings like "Uses do: instead of collect: or select:".

### PR DESCRIPTION
**Pull request by robertkirsten**

### CODE REVIEW

File: packages/Algernon.package/ALGNullItem.class/instance/name.st [hunk 1]
```diff
@@ -1,3 +1,3 @@
 accessing
 name
-^ 'Null Item'
\ No newline at end of file
+	^ 'Null Item'
```
Comment (BenBals) [inline packages/Algernon.package/ALGNullItem.class/instance/name.st:3]: Are you sure there should be a tab here?

Maybe it's just github displaying this weirdly.

File: packages/Algernon.package/ALGResultList.class/instance/hasItems.st [hunk 2]
```diff
@@ -1,3 +1,3 @@
 initialization
 hasItems
-^ self items notEmpty
\ No newline at end of file
+	^ self items notEmpty
```
Comment (BenBals) [inline packages/Algernon.package/ALGResultList.class/instance/hasItems.st:3]: same

File: packages/Algernon.package/ALGResultList.class/instance/items.st [hunk 3]
```diff
@@ -1,3 +1,3 @@
 event handling
 items
-^self submorphs
\ No newline at end of file
+	^ self submorphs
```
Comment (BenBals) [inline packages/Algernon.package/ALGResultList.class/instance/items.st:3]: same

**Review by BenBals — APPROVED** [review]
Looks good to me. :+1: Nice work.

File: packages/Algernon.package/ALGNullItem.class/instance/name.st (hunk 1 above)
Comment (robertkirsten) [reply packages/Algernon.package/ALGNullItem.class/instance/name.st:3]: All other return statements are also indented with a tab, therefore I don't see why there shouldn't be one.

Comment (BenBals) [reply packages/Algernon.package/ALGNullItem.class/instance/name.st:3]: Okay, thank you, then it's just github.

**Comment by BenBals** [comment]
As this is just a bunch of dots, I think we can merge without further review.

**Pull request merged**